### ISSUE TITLE
All map icons are now squares

### ISSUE THREAD

**Issue by oyvindsolstad** [valentine195/obsidian-leaflet-plugin#293]
All marker icons have gone square. Tried the usual (quitting the app, checking for updates).

![Skjermbilde 2022-03-28 kl  18 17 49](https://user-images.githubusercontent.com/4631971/160442736-591f236d-e911-4dd6-955f-7fc74d4409d0.png)

**Comment by valentine195** [comment]
Can you see them in settings?

Also, please let me know your OS and Obsidian version

**Comment by oyvindsolstad** [comment]
Settings:

![Skjermbilde 2022-03-28 kl  18 32 15](https://user-images.githubusercontent.com/4631971/160444895-3cb46fd4-3c1a-4929-b56c-c2f6ee335632.png)

They are good in Settings. MacOS Monterey 12.1 (latest).
Latest public Obsidian version (0.14.2) - which I downloaded and installed between the first post and this (saw that it came out now, this hour). 

Problem exists in newest version too.

**Comment by oyvindsolstad** [comment]
Ok, I opened the code on some of them, changed the ID, and now they seem to be working ok. Nothing wrong with the IDs, I just changed a letter to see if they changed when getting a new ID. 

So seems that did the trick, on all maps I have checked so far.

**Comment by oyvindsolstad** [comment]
Yep, checked 15-20 maps. All are ok now. Consider it closed on my behalf.

![Skjermbilde 2022-03-28 kl  18 46 38](https://user-images.githubusercontent.com/4631971/160447317-37f860c8-38b0-4adf-a6b0-84fab1bb53e9.png)

**Comment by oyvindsolstad** [comment]
(did count, had leaflet on 739 notes !!!)

**Comment by valentine195** [comment]
Do you have Live Preview as your default editing mode?

**Comment by oyvindsolstad** [comment]
I switched from the Legacy editor to the new one, a few days ago. Switching back to test.

**Comment by oyvindsolstad** [comment]
Works perfectly fine with the legacy editor.
